pvlib/scaling.py

```diff
@@ -0,0 +1,320 @@
+"""
+The ``scaling`` module contains functions for manipulating irradiance
+or other variables to account for temporal or spatial characteristics.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def wvm(cs_series, latitude, longitude, cloud_speed,
+        method, capacity=None, density=41, dt=None):
+    """
+    Compute spatial aggregation time series smoothing on clear sky index based
+    on the Wavelet Variability model of Lave et al [1-2]. Implementation is
+    basically a port of the Matlab version of the code [3].
+
+    Parameters
+    ----------
+    cs_series : numeric or pandas.Series
+        Clear Sky Index time series that will be smoothed.
+
+    latitude : numeric
+        Latitudes making up the plant in degrees. Type depends on method used.
+        'discrete' uses a list of lat/lon points representing discrete sites.
+        'square' uses a single valued lat/lon pair for the plant center.
+        'polygon' uses a list of lat/lon points representing the corners of
+        a polygon making up the site.
+
+    longitude : numeric
+        Longitudes making up the plant in degrees. Type depends on method used.
+        'discrete' uses a list of lat/lon points representing discrete sites.
+        'square' uses a single valued lat/lon pair for the plant center.
+        'polygon' uses a list of lat/lon points representing the corners of
+        a polygon making up the site.
+
+    cloud_speed : numeric
+        Speed of cloud movement in meters per second
+
+    method : string
```

Comment:
`plant_type` rather than `method` if we keep the kwarg

Reply:
I vote to ditch `method` plant type/shape in favor of more simplified input [you mention above](https://github.com/pvlib/pvlib-python/pull/807#discussion_r344396077)